### PR TITLE
minor QOL changes in ImageHeader Component

### PR DESCRIPTION
**Pull request by geekysilento**

### CODE REVIEW

File: components/image-header.tsx [hunk 1]
```diff
@@ -29,12 +29,12 @@ export default function ImageHeader({
 
       <section
         className={cn(
-          'absolute flex w-full bg-cover bg-top max-sm:static',
+          'absolute flex w-full bg-neutral-700 bg-cover bg-center bg-blend-overlay max-sm:static',
           'h-56 sm:h-64 md:h-72 lg:h-80 xl:h-96 2xl:h-[448px]',
           className
         )}
         style={{
-          backgroundImage: `url('${getS3Url()}/${src}')`,
```
Comment (GetPsyched) [inline components/image-header.tsx:37]: Uh, this should be temporary if implemented at all. Ideally we'd have `src` mandatory.

If you still wanna add this, keep a FIXME comment to remove sometime later

File: components/image-header.tsx [hunk 2]
```diff
@@ -29,12 +29,12 @@ export default function ImageHeader({
 
       <section
         className={cn(
-          'absolute flex w-full bg-cover bg-top max-sm:static',
+          'absolute flex w-full bg-neutral-700 bg-cover bg-center bg-blend-overlay max-sm:static',
```
Comment (GetPsyched) [inline components/image-header.tsx:32]: The bg will be hidden once the image loads in, right?

Comment (geekysilento) [reply components/image-header.tsx:32]: thats jus a small gradient on top of the image for better text visibililty, like how its done in slideshow

File: components/image-header.tsx [hunk 3]
```diff
@@ -29,12 +29,12 @@ export default function ImageHeader({
...
           'h-56 sm:h-64 md:h-72 lg:h-80 xl:h-96 2xl:h-[448px]',
           className
         )}
         style={{
-          backgroundImage: `url('${getS3Url()}/${src}')`,
+          backgroundImage: `url('${getS3Url()}/${src ? src : 'assets/landingpagebg-1.png'}')`, //fixme: remove this hack once we have good images
```
Comment (GetPsyched) [inline components/image-header.tsx:37]: I don't like to be nitpicky but two things wrong here:
1. Comments start one space after //
2. `FIXME`, `TODO`, etc. are always written in uppercase to draw attention.
```suggestion
          backgroundImage: `url('${getS3Url()}/${src ? src : 'assets/landingpagebg-1.png'}')`, // FIXME: remove this hack once we have good images
```

**Pull request merged**